include/libcyphal/transport/can/transport.hpp

```diff
@@ -372,6 +395,13 @@ class TransportImpl final : public ICanTransport, private TransportDelegate  //
 
     // MARK: Privates:
 
+    template <typename ErrorVariant>
+    CETL_NODISCARD static AnyError anyErrorFromVariant(ErrorVariant&& other_error_var)
+    {
+        return cetl::visit([](auto&& error) -> AnyError { return std::forward<decltype(error)>(error); },
+                           std::forward<ErrorVariant>(other_error_var));
+    }
```

Comment:
This helper nicely converts subset of error types (like `MediaError` alternatives) to the broader `transport::AnyError` variant.